pkg/apis/integration/v1alpha1/integration_types.go

```diff
@@ -33,8 +33,10 @@ type IntegrationSpec struct {
 	Replicas int32 `json:"replicas"`
 	// Docker image consist of micro integrator runtime and synapse configs
 	Image string `json:"image"`
-	// HTTP traffic serving port of the micro integrator runtime
-	Port int32 `json:"port"`
+	// Docker image credentials if the Image is in private registry
+	ImagePullSecret string `json:"imagePullSecret"`
+	// inboundEndointPorts traffic serving port of the micro integrator runtime
+	InboundEndpointPorts []int32 `json:"inboundPorts"`
```

Comment:
I think good to keep the variable name as `inboundPorts`

Reply:
Done